webapp/src/main/webapp/src/app/breadcrumbs/breadcrumbs.component.html

```diff
@@ -1,13 +1,7 @@
 <!-- Breadcrumb -->
-<div [hidden]="breadcrumbs.length == 0">
+<div [hidden]="breadcrumbs.length <= 1">
```

Comment:
If we do actually have a home component later, we would want this to be 0 again, and add back in the home route link I removed below.